decks/contributing-to-ansible.html

```diff
@@ -0,0 +1,434 @@
+<!doctype html>
```

Comment:
Not sure where you would want to put it but I think issues is also a big part of the community. Not just the life cycle but just in general why opening them is good and why working through them and finding fixes is crucial. 

Also just a general theme (which I think you captured well with the title) is making sure people aren't afraid to voice their opinions and also help them understand why their voice is important.

Reply:
* Issues - Good point, created new slide `GitHub Issues and PRs`
* Voice opinions - added in a few places, let me know where else you think I could add this.